docs/spec.md

```diff
@@ -6312,6 +6331,24 @@ function returns `true`. If `x` is not a tensor, returns `None`.
 tensors and returns `num_results` slices of `x` along the axis `axis`.
 If `x` is not a tensor or `dim(x, axis) % num_results != 0`, returns `None`.
 
+* `convert_with_quantized_type(x: Value, destination_type: Type)
```

Comment:
Also, how about moving this definition accordingly, so that all the definitions in this section are written in alphabetical order?